src/lib/targeting.ts

```diff
@@ -1,82 +1,44 @@
-import { EpicTargeting, Tag } from '../components/ContributionsEpic';
+import { EpicTargeting } from '../components/ContributionsEpic';
 
-// Content types allowed to be considered for an Epic
-const ACCEPTED_TYPES = ['Article'];
+const lowValueSections = ['football', 'money', 'education', 'games', 'teacher-network', 'careers'];
```

Comment:
Nice to have this extra context of _why_ we want to blacklist these sections :+1:

Reply:
My $0.02 on this: I don't think we necessarily care _why_ a particular section is whitelisted or blacklisted, not on the Contributions side anyway. We only care whether it's a blacklist of whitelist as that's what the logic around it is all about. We can always add a comment explaining why a section would end up on that list. Today we're blacklisting sections because they have low value but tomorrow it could be because of some legal reason, or any other reason at all. At the end of the day it doesn't matter too much on our side. Right now renaming this isn't a big deal as it's not used anywhere else, but were we to export this to consume on the platform via client lib (e.g. stopping a call being made if the section is blacklisted), we'd potentially have breaking changes. Again, don't think this is too important right now, this is more of a conversation point going forward. It might be useful to think of config variables in more generic terms with name them after the function they serve on our side.